README.md

```diff
@@ -27,21 +27,7 @@ oc apply -f ./setup/serving.yml
 
```

Comment:
under `3. Create the knative-eventing namespace...`

it should be `oc apply -f ./setup/eventing.yml`

Reply:
fixed